Build and Deploy instructions (#5720)
microsoft/MixedRealityToolkit-Unity: Documentation/BuildAndDeploy.md

# Building and Deploying MRTK
When you're ready to run your app on your device a standalone app (for HoloLens, Android, iOS, etc.), you need to build and deploy your project. Building an deploying an app that uses MRTK is just like building and deploying any other Unity app. There are no MRTK-specific instructions. Read below for detailed steps on how to build and deploy a Unity app for HoloLens.  Learn more about building for other platforms at [Publishing Builds](https://docs.unity3d.com/Manual/PublishingBuilds.html).

### Building and Deploying MRTK to HoloLens 1 and HoloLens 2 (UWP)

> As mentioned above, building an app that uses MRTK is just like building any other UWP app, there's nothing MRTK specific. You can follow instructions at [building your application to device](https://docs.microsoft.com/en-us/windows/mixed-reality/mrlearning-base-ch1#build-your-application-to-your-device)  for more information on how to build and deploy your app to HoloLens.

Reviewer: ```suggestion
Instructions on how to build and deploy for Hololens 1 and Hololens 2 (UWP) can be found at [building your application to device](https://docs.microsoft.com/en-us/windows/mixed-reality/mrlearning-base-ch1#build-your-application-to-your-device) .
```

i don't think we have to refer to the previous paragraph , it doesn't add anything here? what do you think?

> **Tip:** When building for WMR, HoloLens 1, or HoloLens 2, it is recommended that your build settings "Target SDK Version"

Reviewer: ```suggestion
**Tip:** When building for WMR, HoloLens 1, or HoloLens 2, it is recommended that the build settings "Target SDK Version"
```

and "Minimum Platform Version" look like they do in the picture below:

> ![](../Documentation/Images/getting_started/BuildWindow.png)

Reviewer: is this something we can easily describe in words so we don't have a screenshot in here that might get outdated?

Reviewer 2: https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/DocumentationGuide.html#images--screenshots

@thalbern I see that you left this comment on a few different places which is just - what do you think about adding your comment to that doc guide section?

Reviewer 2: Looking a bit more, in this case seeing this picture and having folks match it does give some confidence that things are configured correctly... I think this is super easy to mess up and if you do, it's really hard to debug, so I think in this case the value is pretty good.

---
In reply to: [315765611](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5720#discussion_r315765611) [](ancestors = 315765611)

The other settings can be different (for example, Build Configuration/Architecture/Build Type and others can always
be changed inside the Visual Studio solution).

> Make sure that you click on "Target SDK Version" and that it has the option "10.0.18362.0" - if this is missing,

Reviewer: ```suggestion
Make sure that the "Target SDK Version" dropdown includes the option "10.0.18362.0" - if this is missing,
```

> it means you have to install [the latest Windows SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk).

Reviewer: ```suggestion
[the latest Windows SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk) needs to be installed.
```
